### DISCUSSION TITLE
LORIS Release v23.0.3

### DISCUSSION THREAD

**Discussion by driusan** [aces/Loris#7429] [General]
This bugfix release includes all changes from the 22.0.4 release as well as some added bugfixes for the 23.0.2 release.

For the full list of changes **[follow this link.](https://github.com/aces/Loris/pulls?q=is%3Apr+milestone%3A23.0.3+is%3Amerged+)**

Noteworthy changes:
- [Battery] Add missing instruments to a battery when survey instruments are already populated instead of throwing an error #7163
- [Document_repository] Fix for contents displayed at root #7233
- [Filters] Fix Query Param Filtering Bug #7245
- [Instrument_builder] Fix Date element corruption in LINST files when load/saving #7247
- [Instrument] Fix form freeze for completed instruments #7344

Notes for release:
- If upgrading from 23.0.2 to 23.0.3 make sure to run the SQL patch removing the `Email` column from the `participant_accounts` table and the corresponding history entries (see #7248 for more details)
- The bugfix for populating the userId field of instrument tables (#7252) includes a back population script for all userIDs not saved in the instrument tables since this bug appeared. This script will search through the history log to retrospectively populate the userID field. **It's important to note that, depending on the size of your database and history table, the script can take over 15 hours to run.**

<hr /><em>This discussion was created from the release <a href='https://github.com/aces/Loris/releases/tag/v23.0.3'>LORIS Release v23.0.3</a>.</em>

**Comment by jesscall** [comment]
Caveat for projects:
Make sure to run the following scripts in order to remove special characters from links and ensure they are downloadable.
* `tools/single_use/Cleanup_Special_Chars_Media.php` (see #7386)
* `tools/single_use/Cleanup_Special_Chars_Document_Repository.php` (see #7428)

**Reply by driusan** [reply]
Thanks @jesscall, I'll add it to the release notes.
